src/features/surveys/components/surveyForm/SurveySuccess.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box } from '@mui/material';
+import { FC } from 'react';
+import messageIds from 'features/surveys/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import { ZetkinSurveyExtended } from 'utils/types/zetkin';
+
+export type SurveySuccessProps = {
+  survey: ZetkinSurveyExtended;
+};
+
+const SurveySuccess: FC<SurveySuccessProps> = ({ survey }) => {
+  return (
+    <Box>
+      <h1>
+        <Msg id={messageIds.surveyFormSubmitted.title} />
+      </h1>
+      <p>
```

Comment:
I would recommend `<Typography>` instead of `<p>`.

Reply:
Fixed in daaba32837bcb644a4a93be5f1b408a68781361e!